ModManager_Classes/GithubIntegration/GithubDownloader.cs

```diff
@@ -42,7 +44,7 @@ public GithubDownloader(String download_dir)
             }
             catch (Exception e)
             {
-                Console.WriteLine($"Could not fetch Repository Release on {repository.Owner}/{repository.Name}: {e.Message}");
+                throw new InstallationException($"Could not fetch any Repository Release for {repository.Owner}/{repository.Name}: {e.Message}");
```

Comment:
What I like to do is to put constructor into private and make static creation methods that just return null on errors.
Not being able to do IO is pretty normal and it's easy to forget handling exceptions.
The user should get a message box and be able to continue in a clean state instead of a crash in the end.

Reply:
Installation exceptions are caught in InstallerMiddleware.cs, which then reports failure to the InstallationView.